src/main.cpp

```diff
@@ -6,53 +6,77 @@
 #include <inttypes.h>
 #include "esp_log.h"
 
-#include "../sdk/manager/include/manager.hpp"
 #include "../sdk/mock_component/include/mock_component.hpp"
-//#include "../sdk/led_strip/include/led_strip.hpp"
+#include "../sdk/manager/include/manager.hpp"
```

Comment:
We shouldn't be using relative paths, and the path is not correct as sdk has been moved to `../lib/sdk`. ?¿?

Reply:
how is this building